diff --git a/Makefile b/Makefile
index 92edfb9..e3a2675 100644
--- a/Makefile
+++ b/Makefile
@@ -3,6 +3,7 @@ MY_CXXFLAGS := -std=c++14 -O2 -fPIC $(CXXFLAGS)
 MY_CPPFLAGS := -Iznedi3 -Ivsxx $(CPPFLAGS)
 MY_LDFLAGS := $(LDFLAGS)
 MY_LIBS := $(LIBS)
+prefix = /usr/local/Cellar/vsp-znedi3/HEAD-b4a2013
 
 znedi3_HDRS = \
 	znedi3/align.h \
@@ -66,13 +67,20 @@ ifeq ($(X86_AVX512), 1)
   MY_CPPFLAGS := -DZNEDI3_X86_AVX512 $(MY_CPPFLAGS)
 endif
 
-all: vsznedi3.so
+all: vsznedi3.dylib
 
 testapp/testapp: $(testapp_OBJS) $(znedi3_OBJS)
 	$(CXX) $(MY_LDFLAGS) $^ $(MY_LIBS) -o $@
 
-vsznedi3.so: vsznedi3/vsznedi3.o vsxx/vsxx_pluginmain.o $(znedi3_OBJS)
-	$(CXX) -shared $(MY_LDFLAGS) $^ $(MY_LIBS) -o $@
+vsznedi3.dylib: vsznedi3/vsznedi3.o vsxx/vsxx_pluginmain.o $(znedi3_OBJS)
+	$(CXX) -dynamiclib $(MY_LDFLAGS) $^ $(MY_LIBS) -o $@
+
+install: all
+	install AUTHORS.txt $(prefix)
+	install gpl2.txt $(prefix)
+	install readme.rst $(prefix)
+	install vsznedi3.dylib $(prefix)/lib
+	install nnedi3_weights.bin $(prefix)/lib
 
 clean:
 	rm -f *.a *.o *.so testapp/testapp testapp/*.o znedi3/*.o znedi3/x86/*.o vsznedi3/*.o vsxx/*.o
@@ -80,4 +88,4 @@ clean:
 %.o: %.cpp $(znedi3_HDRS) $(testapp_HDRS) $(vsxx_HDRS)
 	$(CXX) -c $(EXTRA_CXXFLAGS) $(MY_CXXFLAGS) $(MY_CPPFLAGS) $< -o $@
 
-.PHONY: clean
+.PHONY: clean install
